src/controllers/organisations.ts

```diff
@@ -227,3 +227,26 @@ export const deleteOrganisation = asyncHandler(async (req, res, next) => {
         data: rows,
     });
 });
+
+/**
+ * @desc    Soft delete single organisation
+ * @route   PUT /api/organisations/:id
+ * @access  Admin/Owner
+ */
+export const deactivateOrganisation = asyncHandler(async (req, res, next) => {
+    const { id } = req.params;
+    const { user_id: userId }: { user_id: string; [key: string]: string } = req.user;
```

Comment:
```suggestion
    const { user_id: userId }: { user_id: string, [key: string]: string } = req.user;
```
Is this a syntax issue?